### ISSUE TITLE
Document that multiple JOSS publications from the same project are possible if substantially different

### ISSUE THREAD

**Issue by matthewfeickert** [openjournals/joss#1307]
In a recent discussion I wanted to share with a team of researchers that they could submit a substantial redesign of their project to JOSS, even though they already have a JOSS publication for it. I was unable to find any documentation on this though, so I wasn't sure where I knew this was possible.

@mstimberg pointed out to me that while this doesn't apprear to be documented yet, there is this 2023 comment from @arfon mentioning it is possible if substantial differences exist between the publication software: https://github.com/openjournals/joss/issues/1162#issuecomment-1427829842

I think it would be useful to have this documented so that we have a clear reference to point authors to who would be interested. Do people agree?

**Comment by arfon** [comment]
> I think it would be useful to have this documented so that we have a clear reference to point authors to who would be interested. Do people agree?

Yes, I think this is a good idea to document. A few things I think we should cover:

- Repeat submissions for the same work are not allowed (should be obvious).
- Follow-up submissions for the same research software are allowed provided:
  - The new release represents a substantial scholarly effort itself.
  - The new release represents a major new release for the software (i.e., I don't think we want 4 publications for 25% of the effort between version 1.0 and version 2.0 of a package, rather we want _one_ publication) – this is sometimes called 'salami-slicing of work'.
  - Submitting authors of the follow-up publications should demonstrate they have alignment/agreement on authorship between the earlier paper and the authors of this new submission. 

I'm least sure about the last one, but it seems like there should be some effort here?
